src/app/app.module.js

```diff
@@ -1,7 +1,31 @@
 import angular from 'angular';
 import appComponent from './app.component';
+import uirouter from '@uirouter/angularjs';
+import helloComp from './hello/hello';
+import worldComp from './world/world';
 import ngMaterial from 'angular-material';
 import ngAnimate from 'angular-animate';
 import ngAria from 'angular-aria';
 
-angular.module('app', [ngMaterial, ngAnimate, ngAria]);
+angular.module('app', [ngMaterial, ngAnimate, ngAria]).component('app', appComponent);
+const myApp = angular.module('app', [uirouter, ngMaterial, ngAnimate, ngAria])
```

Comment:
Create the modules in this way
![image](https://user-images.githubusercontent.com/37654233/57135436-b0864180-6da9-11e9-9165-46026fffc0f8.png)